drivers/net/ethernet/renesas/rswitch_xenback.c

```diff
@@ -58,7 +58,7 @@ static int tx_trigger_time_nsec = 200 * 1000;
 static u16 chain_irq_delay = 0x100;
 module_param(chain_irq_delay, ushort, 0);
 MODULE_PARM_DESC(chain_irq_delay,
-		 "Set IRQ status delay for VMQ device RX/TX chains in range 0-0xfff");
+		 "Set IRQ status delay for VMQ device RX chain in range 0-0xfff");
 
```

Comment:
It looks like smth. compatible with ` ethtool -C|--coalesce`?

Reply:
I have not used this parameter before, but according to ethtool documentation, it seems like it acts the same as R-Switch HW delays.